src/Server/Tasks/Numeral.fs

```diff
@@ -5,12 +5,30 @@ open FSharp.Control.Tasks.V2
 open Giraffe
 open Microsoft.AspNetCore.Http
 
-open Core.Numerals
 open Server.Tasks.Utils
- 
+open Core.Numerals
+open Common.Utils
+open Common.Numerals
+
+let getMinMaxRange = function
+    | From0         -> (0, Int32.MaxValue)
+    | From0To100    -> (0, 100)
+    | From100To1000 -> (100, 1000)
+    | From1000To1000000 -> 
+        (1000, 1000000)
+    | From1000000 -> (1000000, Int32.MaxValue)
+
+let getRandomNumeral (min, max) =
+    Random().Next(min, max)
+
 let getNumeralCardinalsTask next (ctx: HttpContext) =
     task {
-        let number = Random().Next()
+        let range =
```

Comment:
This all works, on the other hand it I am not sure if there is a need for hard ranges there. We might as well just have two fields to specify min and max values and go with it - that would be simpler for us as well. What do you think?

Reply:
Hm, i don't think users will need such granularity for filtering. I assume there few levels that are actually needed, so I tried to make some sense with it

Reply:
Let it be then for now